### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by RahuLDi**
created by rahuld

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,6 +29,7 @@
```
Comment (virazkapoor) [inline index.html]: Don't leave so many spaces in the code

Comment (abhishekk1234) [reply index.html]: chl be

File: index.html [hunk 2]
```diff
@@ -134,3 +144,5 @@
 
 
 
+name: rahuld
```
Comment (as-mis) [inline index.html]: rahulld how are you

Comment (RahuLDi) [reply index.html]: good y say

File: index.html (hunk 1 above)
Comment (abhishekk1234) [reply index.html]: plzzz reply

**Pull request merged**